tests/post_training/test_quantize_conformance.py

```diff
@@ -293,50 +390,49 @@ def test_ptq_quantization(
     capsys: pytest.CaptureFixture,
     extra_columns: bool,
     memory_monitor: bool,
+    use_avx2: Optional[bool] = None,
```

Comment:
use_avx2 used only as fixture to set env variable, no need to pass it to pipeline